src/patternfly/components/Table/table-tree-view.scss

```diff
@@ -157,13 +169,18 @@ $pf-c-tree-view--MaxDepth: 10;
 
   tr:not([hidden]) {
     display: grid;
+    grid-template-columns: 1fr max-content;
   }
 
   // target node rows
   tr[aria-expanded] {
     .pf-c-table__tree-view-title-cell {
       --pf-c-table--cell--PaddingTop: var(--pf-c-table--m-tree-view-grid--tbody--node--PaddingTop);
```

Comment:
Should we update this var to match the one below? Also we could nest the rule below in this block.